site/src/docs/components/accordion/code.mdx

```diff
@@ -90,6 +90,28 @@ import { Accordion } from 'hds-react';
 
 </Playground>
 
+#### Stacked Accordions
+
+<Playground>
+
+```jsx
+import { Accordion } from 'hds-react';
+
+() => (<>
+  <Accordion size="s" card border heading="How to publish data?" language="en" style={{ maxWidth: '360px' }}>
+    To publish your data, open your profile settings and click the button 'Publish'.
+  </Accordion>
+  <Accordion size="s" card border heading="How to publish data?" language="en" style={{ maxWidth: '360px', borderTop: 0 }}>
+    To publish your data, open your profile settings and click the button 'Publish'.
+  </Accordion>
+  <Accordion size="s" card border heading="How to publish data?" language="en" style={{ maxWidth: '360px', borderTop: 0 }}>
+    To publish your data, open your profile settings and click the button 'Publish'.
+  </Accordion>
+</>)
```

Comment:
Maybe we should add this to storybook as well?

Reply:
Would be great to improve the Accordion Card story by adding headings before each accordion to be more descriptive. Card | Small card with borders | Normal card with borders | Large card with borders ?
<img width="465" alt="Screenshot 2023-05-17 at 9 07 30" src="https://github.com/City-of-Helsinki/helsinki-design-system/assets/108321134/84f476f2-1514-42a8-b97e-8719fce48b7d">

Reply:
Ah ok, I'll do those too 👍

Reply:
> Maybe we should add this to storybook as well?

Yeah... and this is what I don't like, keeping two different documentations up-to-date 😭 😆

Reply:
While you're at it, add a proper heading above the accordions to match your other [PR](https://github.com/City-of-Helsinki/helsinki-design-system/pull/1030).